fpsim/locations/senegal/data/method_time_coefficients.csv

```diff
@@ -0,0 +1 @@
+"estimate","coef","se","method","functionform","country"
```

Comment:
@mzimmermann-IDM this file seems to be empty?

Reply:
Oops sorry just fixed it!